### DISCUSSION TITLE
Change the width on the individual chain in the chain transfer

### DISCUSSION THREAD

**Discussion by tiagocunico** [Open-Industry-Project/Open-Industry-Project#80] [Ideas]
On the current release, I'm not able to simulate a real system with the options on the chain transfer configuration.

With this configuration:
![image](https://github.com/user-attachments/assets/8f034068-acc5-4d2b-90db-fb6daf9dda03)
I got this behavior: (the box tilts when been transfer)
![Vídeo sem título ‐ Feito com o Clipchamp](https://github.com/user-attachments/assets/7d1e5144-c038-4968-a826-1c67ddd3560b)

To make it tilt less, I changed the code to be able to build this chain transfer:
![image](https://github.com/user-attachments/assets/fa025983-98df-4928-b5c5-67b5b1d1c1ee)
And got this behavior: 
![TransferenciaOk ‐ Feito com o Clipchamp](https://github.com/user-attachments/assets/5a6bc61e-9d80-4d77-b646-b4ec84f9120b)
It works better, but the problem with this solution is that it is heavily unoptimized, and sometimes the box still tilts when transferring. I think we could solve this problem by been able to increasing the width of the individual chain, so the box doesn't have the space to fall between chains.

**Comment by ryevdokimov** [comment]
https://github.com/Open-Industry-Project/Open-Industry-Project/issues/81

Tracking the issue here.
